### PR TITLE
Update antenna docs

### PR DESCRIPTION
**Pull request by clark2668**
Issue being addressed:

This updates the antenna documentation to fix some mistakes, and add clarifying content.

### CODE REVIEW

File: NuRadioReco/autodoc/pages/detector_database_fields.rst [hunk 1]
```diff
@@ -21,15 +21,84 @@ For dipoles, it can point in any direction that is perpendicular to the first ve
...
+- orientation_phi: orientation of the antenna, as an azimuth angle (counting from East counterclockwise); for LPDA: outward along boresight; for dipoles: upward along axis of azimuthal symmetry
+- rotation_theta: rotation of the antenna, is perpendicular to 'orientation', for LPDAs: vector perpendicular to the plane containing the the tines
+- rotation_phi: rotation of the antenna, is perpendicular to 'orientation', for LPDAs: vector perpendicular to the plane containing the the tines
+
+The user must specify *both* the orientation vector and the rotation vector. 
+
+Here are a few examples.
+
+1. For an LPDA, the following points the LPDA straight down, with tines aligned with
+(parallel to) the x-axis, and therefore pointing eastward.
```
Comment (cg-laser) [inline NuRadioReco/autodoc/pages/detector_database_fields.rst:34]: you mean "pointing northwards" right? The x-axis is along the East-West line, so the vector perpendicular to it is pointing Northwards.

Comment (clark2668) [reply NuRadioReco/autodoc/pages/detector_database_fields.rst:34]: Right, agree. So, the example that I tried to write has the rotation vector along the North/South line, but has the tines along the East/West line. Did I get that wrong?

**Comment by clark2668** [comment]
Sure! I wrote a reply inline above.

File: NuRadioReco/autodoc/pages/detector_database_fields.rst (hunk 1 above)
Comment (cg-laser) [reply NuRadioReco/autodoc/pages/detector_database_fields.rst:34]: No that is correct. Maybe I misunderstood your comment. I though you meant with eastward the direction of the `orientation` vector. Maybe you can rephrase it to 
"1. For an LPDA, the following points the LPDA straight down, with tines aligned with
(parallel to) the x-axis (i.e. the east-west direction), and therefore the orientation vector points northwards."

Comment (clark2668) [reply NuRadioReco/autodoc/pages/detector_database_fields.rst:34]: Yeah, I was trying to talk about the tines, but that tweak adds further clarity. I will make that change!

**Comment by clark2668** [comment]
no problems, will go ahead and merge

**Comment by clark2668** [comment]
oh, actually, I guess I can't because I don't have write permissions. can you do the merge @cg-laser?

**Comment by cg-laser** [comment]
you should have write permissions. Just just need an approving review (which I just did). But we have the policy to wait 24h before we merge so that everyone has the opportunity to comment.

**Comment by clark2668** [comment]
Hmm, the only option it shows me is "close and comment", and it says "Only those with write access to this repository can merge pull requests." So it doesn't seem like I can?

But good point, we will wait till tomorrow in any case!

**Comment by cg-laser** [comment]
Yes you're right. I gave you write access now.
